### ISSUE TITLE
Chatbot "Waiting for Friend" never dismised 

### ISSUE THREAD

**Issue by n8fr8** [zom/Zom-iOS#553]
zombotdev@staging.zom.im is our current dev version of zomkyi bot. When I add it as a friend on iOS, i never get past the "waiting for friend" screen, even though the server side shows an OTR session established, and message seem to get pushed through to Zom iOS.

**Comment by N-Pex** [comment]
Yes, it's not properly subscribed, the bot needs to answer the subscription request. I still see it as "ask=subscribe, subscription=none" in the roster.
